src/components/checkbox/checkbox.component.html

```diff
@@ -12,6 +12,7 @@
                uxFocusIndicator
                class="ux-checkbox-input"
                [id]="(id || _checkboxId) + '-input'"
+               [attr.aria-required]="required"
```

Comment:
This should not be necessary, check the first note here:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute